printer/kubectl_describe.go

```diff
@@ -44,16 +44,17 @@ func (dp *DescribePrinter) Print(r io.Reader, w io.Writer) {
 			key := string(line.Key)
 			if withoutColon, ok := strings.CutSuffix(key, ":"); ok {
 				fmt.Fprint(w, color.Apply(withoutColon, keyColor), ":")
-			} else {
+			} else if !dp.colorFprintLabelOrAnnotation(w, scanner.Path(), key) {
```

Comment:
it's not coloring the keys,  it's parsing  `key` and `value` of annotation/label,  because the annotation/label except first line will be parsed as `line.key`, 
for example:
```
Annotations:  meta.helm.sh/release-name: cert-manager
              meta.helm.sh/release-namespace: nais-system
              some-annotation: true
```
will be pared to:
```
key is [Annotations:], val is [meta.helm.sh/release-name: cert-manager], path is [Annotations]
key is [meta.helm.sh/release-namespace: nais-system], val is [], path is [Annotations/meta.helm.sh/release-namespace: nais-system]
key is [some-annotation: true], val is [], path is [Annotations/some-annotation: true]
```

Reply:
it's not coloring the keys,  it's parsing  `key` and `value` of annotation/label,  because the annotation/label except first line will be parsed as `line.key`, 
for example:
```
Annotations:  meta.helm.sh/release-name: cert-manager
              meta.helm.sh/release-namespace: nais-system
              some-annotation: true
```
will be pared to:
```
key is [Annotations:], val is [meta.helm.sh/release-name: cert-manager], path is [Annotations]
key is [meta.helm.sh/release-namespace: nais-system], val is [], path is [Annotations/meta.helm.sh/release-namespace: nais-system]
key is [some-annotation: true], val is [], path is [Annotations/some-annotation: true]
```

Reply:
Oh, well that feels like a bug in describe.Scanner then

Reply:
Maybe it's ok,  cause such special case  can be fixed in printer